src/sheet_metal_conus.py

```diff
@@ -241,7 +241,7 @@ def effect(self):
         path += self.build_line(dictCone['ptA'], dictCone['ptB'], convFactor) # A,B
         path += " " + self.build_arc(zeroCenter, 0.0, angle, dictCone['longRadius']*convFactor)
         path += " " + self.build_line(dictCone['ptC'], dictCone['ptD'], convFactor) # C,D
-        path += self.build_arc(zeroCenter, angle, 0.0, dictCone['shortRadius']*convFactor, False)
+        path += self.build_arc(zeroCenter, 0.0, angle, dictCone['shortRadius']*convFactor)
```

Comment:
fixing an error where sometimes wrong arc will be drawn. so explicitly setting direction of arc.